src/Features/Core/Portable/Diagnostics/DiagnosticAnalyzerService_IncrementalAnalyzer.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.CodeAnalysis.Diagnostics
 {
     [ExportIncrementalAnalyzerProvider(
         highPriorityForActiveFile: true, name: WellKnownSolutionCrawlerAnalyzers.Diagnostic,
-        workspaceKinds: new string[] { WorkspaceKind.Host, WorkspaceKind.Interactive, WorkspaceKind.CloudEnvironmentClientWorkspace })]
+        workspaceKinds: new string[] { WorkspaceKind.Host, WorkspaceKind.Interactive })]
```

Comment:
This doesn't break diagnostics in some way?

Reply:
No, this used to be there b/c the roslyn diagnostic service was running in the client (and calling the server to get diagnostics).  Now that we're using LSP diagnostics it doesn't need to run on the client.